DiceRoller/androidApp/build.gradle.kts

```diff
@@ -36,6 +36,28 @@ android {
         compose = true
         buildConfig = false
     }
+    buildTypes {
+        getByName("release") {
+            // Enables code shrinking, obfuscation, and optimization for only
+            // your project's release build type. Make sure to use a build
+            // variant with `isDebuggable=false`.
+            isMinifyEnabled = true
+
+            // Enables resource shrinking, which is performed by the
+            // Android Gradle plugin.
+            isShrinkResources = true
+
+            proguardFiles(
+                // Includes the default ProGuard rules files that are packaged with
+                // the Android Gradle plugin. To learn more, go to the section about
+                // R8 configuration files.
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+
+                // Includes a local, custom Proguard rules file
+                "proguard-rules.pro"
```

Comment:
do we have this file?

Reply:
I created one for DiceRoller. If it's not needed here we can remove this line.

Reply:
do you mean you created one for Fruitties, Ben?

Reply:
Yes, that's what I meant.